### ISSUE TITLE
Using npmregistries for private packages, is failing to load package. Because of CORS error.

### ISSUE THREAD

**Issue by appanasantosh** [codesandbox/sandpack#553]
# Bug report

## Packages affected

-   [ ] sandpack-client
-   [Y] sandpack-react

### Used npmregistry for loading packages from custom registry

 npmRegistries: [
          {
            proxyEnabled: true,
            enabledScopes: ['@scope'],
            limitToScopes: true,
            registryUrl: "custom registry url",
            registryAuthToken:"token"
          },
        ],

Error info: 'https://1-4-0-sandpack.codesandbox.io' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'.

### Your Environment

| Software                | Name/Version |
| ----------------------- | ------------ |
| Sandpack-client version |              |
| Sandpack-react version  |        1.5      |
| Browser                 |     chrome         |
| Operating System        |     windows10         |

**Comment by danilowoz** [comment]
Hey @appanasantosh, thanks for reporting this issue. However, given the amount of details in this description, I can't really debug and understand what's happening. Please, instead of mentioning a piece of documentation, give a real snippet of code (hide sensitive information) and add more information, such as: a network screenshot, sandpack error, or anything else that make it easier to understand the problem.

Feel free to reach out to me on Discord, and then we can make a session to debug it.

**Comment by appanasantosh** [comment]
Hi @danilowoz here is the screenshot of the error when trying to import the private registry package.
![image](https://user-images.githubusercontent.com/10960803/185613486-eb352662-4686-498a-82cf-a8feb5c4c3b3.png)

In the code I have mentioned the details about the registry in prop 
npmRegistries: [
{
proxyEnabled: true,
enabledScopes: ['https://github.com/scope'],
limitToScopes: true,
registryUrl: "custom registry url",
registryAuthToken:"token"
},
]

**Comment by DeMoorJasper** [comment]
Most registries don't add the cors headers, so unless you have a proxy like `verdaccio` in place or have added cors headers to your registry in some other way, it will return this error.

Don't think there's a lot we can do here unless you share more details about the registry itself as the issue is with the registry not sandpack.

**Comment by appanasantosh** [comment]
Actually the registries are from private azure artifacts space.

**Comment by danilowoz** [comment]
@appanasantosh, the configuration should look something like this:
```js
npmRegistries: {
  enabledScopes: ["@codesandbox"],
  registryUrl: "//my-registry.domain.com",
  limitToScopes: true,
  registryAuthToken: "SECRET" // optional value, if public
}
```

`proxyEnabled` must be `false` and the `enableScopes` must contain the package scope instead of an URL.

**Comment by appanasantosh** [comment]
I have used the same configuration before, but the same exception was thrown as mentioned in above comments.
![image](https://user-images.githubusercontent.com/10960803/185845034-380c760d-9409-4356-b7e5-9c45bb62402d.png)

**Comment by danilowoz** [comment]
Hey @appanasantosh, indeed, there was a CORS issue on the latest bundler version. It has been fixed on the latest Sandpack version and https://github.com/codesandbox/sandpack/pull/554

**Comment by appanasantosh** [comment]
@danilowoz I have installed latest version(1.5.2) and tested it. Still I am having the same issue.

**Comment by danilowoz** [comment]
Can you try again with 1.5.3? It should work now

**Comment by danilowoz** [comment]
Hey, it should be fixed on v1.5.4, thanks again for reporting it 👍

**Comment by appanasantosh** [comment]
Thanks @danilowoz. CORS issue is fixed, it is working fine now.
